packages/pds/package.json

```
@@ -48,7 +48,7 @@
     "bytes": "^3.1.2",
     "compression": "^1.7.4",
     "cors": "^2.8.5",
-    "disposable-email-domains-js": "^1.5.0",
+    "disposable-email-domains-js": "^1.6.0",
```

```suggestion
    "disposable-email-domains-js": "^1.8.0",
```
The latest version is now `1.8.0` so it probably makes sense to update to that.